### PR TITLE
Hentai20 Theme Changed Fix

### LINKED ISSUE #718: Hentai20 extension is broken
### Source information

Hentai20 1.4.36

### Source language

English

### Steps to reproduce

1. Download hentai20 extension
2. Trust it
3. Open the extension it will not load any series

### Expected behavior

It should load the series

### Actual behavior

It dosn't load any series and show no result found and you can't update any series while the webview works fine

### Tachiyomi version

Tachiyomi 0.15.3 and miho 0.16.1

### Android version

Android 10

### Other details

Its not a cloudflare protection problem

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by devil6venom**
Checklist:
Closes: #718

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [ ] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [ ] Have tested the modifications by compiling and running the extension through Android Studio
- [ ] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

File: multisrc/overrides/madara/hentai20/src/Hentai20.kt [hunk 1]
```diff
@@ -1,13 +0,0 @@
...
-import eu.kanade.tachiyomi.multisrc.madara.Madara
-import eu.kanade.tachiyomi.network.interceptor.rateLimit
-import okhttp3.OkHttpClient
-import java.util.concurrent.TimeUnit
-
-class Hentai20 : Madara("Hentai20", "https://hentai20.io", "en") {
-
-    override val client: OkHttpClient = super.client.newBuilder()
-        .rateLimit(1, 1, TimeUnit.SECONDS)
-        .build()
```
Comment (beer-psi) [inline multisrc/overrides/madara/hentai20/src/Hentai20.kt:12]: Why didn't you preserve the old rate limit?

**Review by beer-psi — CHANGES_REQUESTED** [review]

File: multisrc/overrides/madara/hentai20/src/Hentai20.kt (hunk 1 above)
Comment (devil6venom) [reply multisrc/overrides/madara/hentai20/src/Hentai20.kt:12]: Oh i am sorry. I thought maybe in Mangathemesia it doesn't use it..as i checked perviously.. it was working fine. Let me add that! Better to be careful!

Comment (devil6venom) [reply multisrc/overrides/madara/hentai20/src/Hentai20.kt:12]: > Why didn't you preserve the old rate limit?

Say should i add it in src of hentai20 or ?? Please help me!

File: multisrc/overrides/mangathemesia/hentai20/src/hentai20.kt
Comment (beer-psi) [inline multisrc/overrides/mangathemesia/hentai20/src/hentai20.kt:1]: The file name should match the class name, case-sensitive. Rename to `Hentai20.kt` with a capital H.

**Review by beer-psi — CHANGES_REQUESTED** [review]

File: multisrc/overrides/mangathemesia/hentai20/src/hentai20.kt
Comment (devil6venom) [reply multisrc/overrides/mangathemesia/hentai20/src/hentai20.kt:1]: Oh okay

**Review by beer-psi — APPROVED** [review]

**Pull request merged**